### PR TITLE
Add PDB for eck-operator helm chart

### PR DESCRIPTION
**Pull request by eliskovets**
Allow PodDisruptionBudget be configured via parameter for eck-operator statefulset deployment.

### CODE REVIEW

File: deploy/eck-operator/values.yaml [hunk 1]
```diff
@@ -68,6 +68,12 @@ tolerations: []
 # affinity defines the node affinity rules for the operator pod.
 affinity: {}
 
+# Only one of minAvailable or maxUnavailable can be set
+podDisruptionBudget:
+  enabled: false
+  # minAvailable: 1
```
Comment (thbkrkr) [inline deploy/eck-operator/values.yaml:74]: I would set by default `enabled: false` and `minAvailable: 1` (not commented).
```suggestion
  minAvailable: 1
```

**Comment by eliskovets** [comment]
@thbkrkr Thank you for all the suggestions. I've updated the PR.

**Comment by thbkrkr** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by thbkrkr** [comment]
buildkite test this

**Review by thbkrkr — APPROVED** [review]

**Pull request merged**